### DISCUSSION TITLE
HMiP-RFUSB auf arch linux

### DISCUSSION THREAD

**Discussion by j0chn1** [OpenCCU/OpenCCU#2122] [🇩🇪 Fragen&Antworten] [answered]
Hallo Zusammen,
ich habe in der Doku dazu leider keine konkrete Antwort gefunden und auch google hat nicht wirklich was gebracht,
Wenn ich mir den Funk-USB Stick hole, dann kann ich den an jedem Gerät anstecken und nutzen, wenn ich eine VM mit Raspberrymatic laufen lassen.
Aber es steht nicht explizit, dass es nicht möglich ist irgendwelche packages zu installieren und das dann "nativ" laufen zu lassen. Und wenn ich das richtig sehe, ist Raspberrymatic auf Linux basierend. Somit wäre es doch thereotsich mögich, oder?
Ich wollte mein thin client nutzen, auf dme Arch Linux installiert ist, da der mehr Power hat, als der Raspberry. 
Kann ich sonst auf dem RaspberryMatic image noch andere Sachen installieren (z.B. PiHole)?
Danke für die Antworten.

**Comment by Baxxy13** [comment, marked as answer]
Wenn du mehr als RaspberryMatic auf einem Hostsystem laufen lassen willst ist eine Virtualisierungslösung (wie z.B. Proxmox) zu empfehlen.
RaspberryMatic im Docker-Container geht auch. Muss man aber mögen und man sollte sich mit Docker auskennen.

Alternativen wären piVCCU oder DebMatic.

Wenn RaspberryMatic direkt auf dem Host läuft lässt sich nichts zusätzliches installieren. Dafür ist das von RaspberryMatic genutzte Linux nicht gemacht.

**Reply by j0chn1** [reply]
Dann schaue ich lieber, dass ich die 1 oder 2 services com Raspy auf das andere System ziehe. Hab noch keine VM Lösung unter Linux zum laufen bekommen.
Offiziell wird DSR Raspy 4 nicht unterstützt. Könnt ihr sagen, ob 
Das HM-MOD-RPI-PCB
Eine sichere Investition ist?
Würde mir das dann mit verbesserter Antenne zulegen wollen.
Allerdings fertig gebaut und geflasht. Bin da weniger versiert...

**Reply by Baxxy13** [reply]
> HM-MOD-RPI-PCB

Genau dieses Funkmodul kann man aktuell nicht mehr empfehlen.
Damit kann man z.B. kein HmIP-Wired nutzen, keinen HAP als LAN-Router anbinden und man sieht auch mögliche Funkstörer (CARRIER_SENSE) nicht.
Nimm lieber gleich das RPI-RF-MOD oder den HmIP-RFUSB.

Gelötet werden müssen alle (außer du kaufst eine CCU3), flashen muss man keines (das geht automatisch).

**Reply by j0chn1** [reply]
Hab jetzt das RPI-RF-MOD und hat alles geklappt. Danke !!!
